addons/mail/static/src/core/common/composer.xml

```diff
@@ -14,7 +14,7 @@
                     'p-3': normal,
                     'o-hasSelfAvatar': !env.inChatWindow and thread,
                 }" t-attf-class="{{ props.className }}">
-            <div class="o-mail-Composer-sidebarMain flex-shrink-0" t-if="!compact and props.sidebar">
+            <div class="o-mail-Composer-sidebarMain flex-shrink-0" t-if="!compact and props.sidebar" name="sidebar main">
```

Comment:
name is not really necessary in this case, it could target the class `o-mail-Composer-sidebarMain`.